### PR TITLE
[#3509] Fix address server mode cannot be obtained application.properties

### PR DESCRIPTION
**Pull request by qm0123**
#  What is the purpose of the change

see #3509 

## Brief changelog

1. Use ApplicationUtils.getProperty () get variable

2. Synchronize the parameters in properties

3. After the change, get the priority system environment variable of the variable, then start the JVM (- D) - > properties property

### CODE REVIEW

**Comment by chuntaojun** [comment]
请按照贡献文档对你的代码进行格式化下

File: core/src/main/java/com/alibaba/nacos/core/cluster/lookup/AddressServerMemberLookup.java [hunk 1]
```diff
@@ -43,30 +43,30 @@
  * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
  */
 public class AddressServerMemberLookup extends AbstractMemberLookup {
-    
+
```
Comment (KomachiSion) [inline core/src/main/java/com/alibaba/nacos/core/cluster/lookup/AddressServerMemberLookup.java:46]: Do not change indent.
Please use nacos code style to format.
see src/style/codeStyle.md

Comment (qm0123) [reply core/src/main/java/com/alibaba/nacos/core/cluster/lookup/AddressServerMemberLookup.java:46]: ok,copy that.

File: distribution/conf/application.properties [hunk 2]
```diff
@@ -37,8 +37,8 @@ server.port=8848
 
 ### Connect URL of DB:
 # db.url.0=jdbc:mysql://127.0.0.1:3306/nacos?characterEncoding=utf8&connectTimeout=1000&socketTimeout=3000&autoReconnect=true&useUnicode=true&useSSL=false&serverTimezone=UTC
-# db.user=nacos
-# db.password=nacos
+# db.user=root
```
Comment (KomachiSion) [inline distribution/conf/application.properties:40]: do not change the default value

File: distribution/conf/application.properties [hunk 3]
```diff
@@ -37,8 +37,8 @@ server.port=8848
...
+# db.password=1017
```
Comment (KomachiSion) [inline distribution/conf/application.properties:41]: do not change the default value

File: distribution/conf/application.properties.example [hunk 4]
```diff
@@ -29,16 +29,16 @@ server.port=8848
...
 # spring.datasource.platform=mysql
 
 ### Count of DB:
 # db.num=1
 
 ### Connect URL of DB:
 # db.url.0=jdbc:mysql://127.0.0.1:3306/nacos?characterEncoding=utf8&connectTimeout=1000&socketTimeout=3000&autoReconnect=true&useUnicode=true&useSSL=false&serverTimezone=UTC
-# db.user=nacos
-# db.password=nacos
+# db.user=root
```
Comment (KomachiSion) [inline distribution/conf/application.properties.example:40]: do not change the default value

File: distribution/conf/application.properties.example [hunk 5]
```diff
@@ -29,16 +29,16 @@ server.port=8848
...
+# db.password=1017
```
Comment (KomachiSion) [inline distribution/conf/application.properties.example:41]: do not change the default value

File: distribution/conf/application.properties (hunk 2 above)
Comment (qm0123) [reply distribution/conf/application.properties:40]: I think it should be combined with (console-src-main-resources-application.properties)The default value of is the same, but nacos:nacos Is the default value, I will change the default value to nacos:nacos

File: distribution/conf/application.properties (hunk 3 above)
Comment (qm0123) [reply distribution/conf/application.properties:41]: I think it should be combined with (console-src-main-resources-application.properties)The default value of is the same, but nacos:nacos Is the default value, I will change the default value to nacos:nacos

File: distribution/conf/application.properties.example (hunk 4 above)
Comment (qm0123) [reply distribution/conf/application.properties.example:40]: I think it should be combined with (console-src-main-resources-application.properties)The default value of is the same, but nacos:nacos Is the default value, I will change the default value to nacos:nacos

File: distribution/conf/application.properties.example (hunk 5 above)
Comment (qm0123) [reply distribution/conf/application.properties.example:41]: I think it should be combined with (console-src-main-resources-application.properties)The default value of is the same, but nacos:nacos Is the default value, I will change the default value to nacos:nacos

**Pull request merged**